soccer/gameplay/plays/defense/advanced_pileup.py

```diff
@@ -0,0 +1,91 @@
+import robocup
+import constants
+import play
+import skills
+import tactics
+import main
+import evaluation.position
+import math
+import random
+
+class AdvancedPileup(play.Play):
```

Comment:
This will only execute when this play starts, after that first frame this starts, the positions don't get updated. Is that the goal?

Reply:
Since this is only run when the pileup begins, I am assuming that the pileup does not move in any way during its duration, and just getting the original positions to prepare for collection when the ball is out of the pileup.